UPGRADE-2.8.md

```diff
@@ -238,6 +238,21 @@ Form
 Translator
 ----------
 
+ * Loading catalogues (file caching, resource loader, locale fallback) has been moved out of Translator and the class
+   `Symfony\Bundle\FrameworkBundle\Translation\Translator` was deprecated and will be removed in Symfony 3.0.
+
+   If you want to use the new dedicated translator service, you should either
+   explicitly define "translator" service.
```

Comment:
you should either define the `translator` service or what? (usually you say "either do X or do Y")?

And is the `translator` service now gone? Why and when would I do the below stuff?